src/pages/CartPage.jsx

```diff
@@ -0,0 +1,13 @@
+import { Suspense } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { getCart } from '../services/cart';
+import Loader from '../components/atoms/Loader';
+import CartList from '../components/molecules/CartList';
+
+const CartPage = () => {
+    const { data } = useQuery(['cart'], () => getCart());
```

Comment:
```suggestion
    const { data } = useQuery(['cart'], getCart);
```

처럼 간략하게 작성할 수 있을 것 같습니다!
react-query 5부터는 객체 형태의  API만 혀용되고 있습니다.